libs/tf2-format/src/lib/sku/sku.ts

```diff
@@ -22,7 +22,7 @@ export class SKU {
     if (item.wear) {
       sku += `;w${item.wear}`;
     }
-    if (item.paintkit) {
+    if (item.paintkit !== null) {
```

Comment:
It should be checking if the type is number because the value could be null or undefinded.